### PR TITLE
PLANNER-551 : Workbench domain editor: dock tab icon should use OptaPlanner icon (current icon has no relation to planning optimization)

### PR DESCRIPTION
**Pull request by mcimbora**

### CODE REVIEW

File: optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/resources/images/AppImageResources.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+package org.optaplanner.workbench.client.resources.images;
+
+import com.google.gwt.resources.client.ClientBundle;
+import com.google.gwt.resources.client.ImageResource;
+
+public interface AppImageResources extends ClientBundle {
+
+    @Source("optaPlannerIconBlue.png")
+    ImageResource optaPlannerIconBlue();
```
Comment (ge0ffrey) [inline optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/resources/images/AppImageResources.java]: @csadilek What's the naming convention guideline here: name them as how they look or as what purpose they do? So optaPlannerBlueIcon.png versus optaPlannerHoverOverIcon.png (or optaPlannerEnabledIcon.png)?

Comment (csadilek) [inline optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/resources/images/AppImageResources.java]: @ge0ffrey optaPlannerIcon, optaPlannerEnabledIcon optaPlannerDisabledIcon are all fine. I wouldn't use the color etc. (how things look) as part of the name as you should be able to update or replace images without changing the resource method name...

Comment (ge0ffrey) [inline optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/resources/images/AppImageResources.java]: @mcimbora ^^ I got the same opinion.

Comment (mcimbora) [inline optaplanner-wb-webapp/src/main/java/org/optaplanner/workbench/client/resources/images/AppImageResources.java]: @csadilek @ge0ffrey Thanks for the review, fixed.

**Comment by ge0ffrey** [comment]
+1 to merge

**Pull request merged**